lms/tasks/email_digests.py

```diff
@@ -60,13 +61,19 @@ def send_instructor_email_digest_tasks(batch_size):
 
             for batch in batches:
                 send_instructor_email_digests.apply_async(
-                    [batch],
-                    {"updated_after": updated_after, "updated_before": updated_before},
+                    (),
+                    {
+                        "h_userids": batch,
+                        "updated_after": updated_after,
+                        "updated_before": updated_before,
+                    },
```

Comment:
I changed `send_instructor_email_digests()` to use keyword-only arguments (see below) so this is just updating the call here to use kwargs.